src/preview-app.html

```diff
@@ -146,6 +153,7 @@
             </paper-tab>
           </paper-tabs>
         </template>
+        <paper-button raised on-tap="_signout" hidden$="[[_showSignOut(page.page)]]">Sign out</paper-button>
```

Comment:
I would prefer if the button is not `raised`, but instead changes color on hover.